packaging/faucet/miden-faucet.service

```diff
@@ -7,8 +7,8 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
-Environment="RUST_LOG=info"
-ExecStart=/usr/bin/miden-faucet start --config /etc/opt/miden-faucet/miden-faucet.toml
+EnvironmentFile=/lib/systemd/system/miden-faucet.env
```

Comment:
How does this interact ito overriding values?

We install the debian package, which also installs this env file. How does one now override these default values?

Reply:
You can run `systemd-edit` which will create an overrides file in i.e. `/etc/systemd/system/miden-faucet.service.d/override.conf` which is used to patch the original.